src/Common.jl

```diff
@@ -330,6 +335,39 @@ function Chen2022_exponential_pdf(a::FT, b::FT, c::FT, λ_inv::FT, k::Int) where
     return a * exp(δ * log(1 / λ_inv) - (b + δ) * log(1 / λ_inv + c)) * SF.gamma(b + δ) / SF.gamma(δ)
 end
 
+"""
+    liquid_particle_terminal_velocity(velocity, ρₐ)
+    liquid_particle_terminal_velocity(velocity, ρₐ, D)
+
+Compute the terminal velocity of a liquid particle as a function of its size 
+    (maximum dimension, `D`) using the Chen 2022 parametrization.
+
+# Arguments
+- `velocity`: a struct with terminal velocity parameters from Chen 2022
```

Comment:
I would vote to keep the naming convention of params, coefficients, etc. Maybe `velocity_params` or something like that if you want to be more precise?